### PR TITLE
Displaying valid error based on error-tag, even if error-message attribute is empty

### PR DESCRIPTION
**Pull request by justinejose91**

### CODE REVIEW

**Review by dutta-partha — APPROVED** [review]
Sachin Holla to confirm if we need to handled all error-tags as below :

•         invalid-value 
•         operation-failed
•         operation-not-supported
•         access-denied
•         resource-denied
•         in-use
•         malformed-message

File: src/CLI/klish/patches/klish-2.1.4/plugins/clish/rest_cl.cpp [hunk 1]
```diff
@@ -106,10 +106,26 @@ int print_error(const char *str) {
         cJSON *err_msg = cJSON_GetObjectItemCaseSensitive(error, "error-message");
         if (err_msg) {
             lub_dump_printf("%% Error: %s\r\n", err_msg->valuestring);
+        /* Since error-message is an optional attribute, we need to check for "error-tag"
+           and print the error-message accordingly */
+        } else {
+            cJSON* err_tag = cJSON_GetObjectItemCaseSensitive(error, "error-tag");
+            std::string err_tag_str = std::string {err_tag->valuestring};
```
Comment (sachinholla) [inline src/CLI/klish/patches/klish-2.1.4/plugins/clish/rest_cl.cpp:119]: REST server always populates error-tag in the error response. Still, I feel it is better to have a null check for err_tag.

Comment (justinejose91) [reply src/CLI/klish/patches/klish-2.1.4/plugins/clish/rest_cl.cpp:119]: That's a good one. I will add that check. I missed it somehow 👍 Fixed it.

**Comment by justinejose91** [comment]
Screen Shot 2020-02-06 at 7 13 10 PM

**Review by sachinholla — APPROVED** [review]

**Pull request merged**